src/libraries/Microsoft.PowerFx.Core/IR/InvalidCoercionException.cs

```diff
@@ -11,5 +11,14 @@ public InvalidCoercionException(string message)
             : base(message)
         {
         }
+
+        public InvalidCoercionException()
```

Comment:
Why add an empty constructor?

Reply:
This matters - C# won't automatically create a default ctor if there are already other ctors present. 

So before, 
new InvalidCoercionException() // illegal, must call a required ctor

Reply:
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1032](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1032)